spec/intro.md

```diff
@@ -78,10 +78,6 @@ A reference to a _term_ looks like this.
 
 ### Stability Policy
 
-> [!IMPORTANT]
-> The provisions of the stability policy are not in effect until
-> the conclusion of the technical preview and adoption of this specification.
-
 Updates to this specification will not make any valid _message_ invalid.
 
 Updates to this specification will not remove any syntax provided in this version.
```

Comment:
"Remove" any syntax is ambiguous. That would appear to prevent us from changing %XX / %YY / %ZZ to %XX-%ZZ.

It should be something like (but with better wording):
```suggestion
Updates to this specification will not make any message not match, where that message had matched a previous version's syntax.
```

Reply:
ok.

> Updates to this specification will not change the syntactical meaning
of any syntax defined in this specification.	of any syntax defined in this specification.

We need room to be able to extend and/or clarify the meaning. And how do we even test whether 'syntactical meaning' have changed.  I think it is redundant. If it remains, it needs to be restated in terms of compatibility. Maybe like: 
> Updates to this specification will not change the syntactical meaning
of any syntax defined in this specification in a way that makes messages valid according to previous versions invalid.

Reply:
I propose we kill this item.

Reply:
Agreed!